### PR TITLE
Adding the {year} option to InputInfo 

### PR DESCRIPTION
**Pull request by vukasinmilosevic**
A quick PR to add the {year} option to InputInfo @ram1123

### CODE REVIEW

File: efficiencyFactors.py [hunk 1]
```diff
@@ -961,10 +967,10 @@ def geteffs(channel, SampleList, m4l_bins, m4l_low, m4l_high, obs_reco, obs_gen,
     f.write('lambdajesdn = '+str(lambdajesdn)+' \n')
 
 
-more_output_file_name = datacardInputs+'/moreinputs_sig_'+opt.OBSNAME+ext+'.py'
+more_output_file_name = opt.ERA+'/'+datacardInputs+'/moreinputs_sig_'+opt.OBSNAME+ext+'.py'
```
Comment (ram1123) [inline efficiencyFactors.py:970]: I think here we don't need `opt.ERA`.

```python
datacardInputs.format(year = opt.ERA)
```

File: efficiencyFactors.py [hunk 2]
```diff
@@ -961,10 +967,10 @@ def geteffs(channel, SampleList, m4l_bins, m4l_low, m4l_high, obs_reco, obs_gen,
...
 
 if not (obs_reco2 == ''):
-    more_output_file_name = datacardInputs+'/moreinputs_sig_'+opt.OBSNAME.replace(" ", "_")+'.py'
+    more_output_file_name = opt.ERA+'/'+datacardInputs+'/moreinputs_sig_'+opt.OBSNAME.replace(" ", "_")+'.py'
```
Comment (ram1123) [inline efficiencyFactors.py:973]: same previous comment?

File: getUnc_Unc.py [hunk 3]
```diff
@@ -51,6 +52,7 @@ def parseOptions():
 
 if (not os.path.exists("plots") and doPlots):
     os.system("mkdir plots")
+    os.system('mkdir plots/'+opt.ERA)
```
Comment (ram1123) [inline getUnc_Unc.py:55]: If I remember correctly, here the plotter is not working. We need to also fix this.

File: Inputs/Input_Info.py [hunk 4]
```diff
@@ -7,7 +7,8 @@
 
 ################################
 # Name of output directories
-datacardInputs = "datacardInputs"
+datacardInputs = "{year}/datacardInputs"
```
Comment (ram1123) [inline Inputs/Input_Info.py:10]: If we add {year} here, then we need to propagate this info to the efficiency module

```bash
Traceback (most recent call last):
  File "efficiencyFactors.py", line 947, in <module>
    if not os.path.isdir(opt.ERA+"/"+datacardInputs): os.mkdir(opt.ERA+"/"+datacardInputs)
OSError: [Errno 2] No such file or directory: '2018/{year}/datacardInputs'
```

**Comment by vukasinmilosevic** [comment]
I think I've misunderstood what {year} will add to the overall code, I assumed it was only for the collectInputs. I will close this one and propagate it to the rest of the code

**Pull request closed without merging**